config/environments/production.rb

```diff
@@ -1,4 +1,4 @@
-require "active_support/core_ext/integer/time"
+require 'active_support/core_ext/integer/time'
 
 Rails.application.configure do
```

Comment:
While you did a good job working on the project there is an important aspect you missed out in your database yml file. You are using the wrong database adapter please take a look at the screenshot below:
![Screenshot 2023-11-04 at 12 11 10 PM](https://github.com/firstpj/blog-app/assets/79506105/f56c3c56-dd5b-4186-aff8-e69dc398a4b8)

you used sqlite3 as your adapter the correct adapter to use is this:
default: &default
adapter: postgresql
encoding: unicode

...